console/services/app.py

```diff
@@ -713,6 +714,16 @@ def create_app_market(self, data):
             raise ServiceHandleException(msg="name exist", msg_show=u"名称已存在", status_code=400)
         return app_market_repo.create_app_market(**data)
 
+    @transaction.atomic
+    def batch_create_app_market(self, eid, data):
+        for dt in data:
```

Comment:
You should first determine that the data type and value are not None.

Reply:
The 'data' comes from the upper level function, and the upper level function has initialized the 'data' as an empty list , So we also need to judge the data type and value are not None?

Reply:
Of course, the caller is untrustworthy.